docs/_data/MIPoPS_Handbrake_Access.json

```diff
@@ -0,0 +1,115 @@
+{
+  "PresetList" : [
+    {
+      "AlignAVStart" : true,
+      "AudioCopyMask" : [
+        "copy:aac"
+      ],
+      "AudioEncoderFallback" : "ca_aac",
+      "AudioLanguageList" : [
+
+      ],
+      "AudioList" : [
+        {
+          "AudioBitrate" : 160,
+          "AudioCompressionLevel" : -1,
+          "AudioDitherMethod" : "auto",
+          "AudioEncoder" : "ca_aac",
+          "AudioMixdown" : "stereo",
+          "AudioNormalizeMixLevel" : false,
+          "AudioSamplerate" : "auto",
+          "AudioTrackDRCSlider" : 0,
+          "AudioTrackGainSlider" : 0,
+          "AudioTrackQuality" : -1,
+          "AudioTrackQualityEnable" : false
+        }
+      ],
+      "AudioSecondaryEncoderMode" : true,
+      "AudioTrackSelectionBehavior" : "first",
+      "ChapterMarkers" : true,
+      "ChildrenArray" : [
+
+      ],
+      "Default" : false,
+      "FileFormat" : "av_mp4",
+      "Folder" : false,
+      "FolderOpen" : false,
+      "InlineParameterSets" : false,
+      "Mp4HttpOptimize" : true,
+      "Mp4iPodCompatible" : false,
+      "PictureAutoCrop" : false,
+      "PictureBottomCrop" : 0,
+      "PictureCombDetectCustom" : "",
+      "PictureCombDetectPreset" : "default",
+      "PictureDARWidth" : 0,
+      "PictureDeblock" : 0,
+      "PictureDeblockCustom" : "qp=0:mode=2",
+      "PictureDeinterlaceCustom" : "",
+      "PictureDeinterlaceFilter" : "decomb",
+      "PictureDeinterlacePreset" : "default",
+      "PictureDenoiseCustom" : "",
+      "PictureDenoiseFilter" : "nlmeans",
+      "PictureDenoisePreset" : "ultralight",
+      "PictureDenoiseTune" : "tape",
+      "PictureDetelecine" : "off",
+      "PictureDetelecineCustom" : "",
+      "PictureForceHeight" : 0,
+      "PictureForceWidth" : 0,
+      "PictureHeight" : 0,
+      "PictureItuPAR" : false,
+      "PictureKeepRatio" : true,
+      "PictureLeftCrop" : 0,
+      "PictureLooseCrop" : false,
+      "PictureModulus" : 2,
+      "PicturePAR" : "auto",
+      "PicturePARHeight" : 10,
+      "PicturePARWidth" : 9,
+      "PictureRightCrop" : 0,
+      "PictureRotate" : "angle=0:hflip=0",
+      "PictureSharpenCustom" : "",
+      "PictureSharpenFilter" : "off",
+      "PictureSharpenPreset" : "medium",
+      "PictureSharpenTune" : "none",
+      "PictureTopCrop" : 0,
+      "PictureWidth" : 0,
+      "PresetDescription" : "",
+      "PresetName" : "MIPoPS_Access",
+      "SubtitleAddCC" : false,
+      "SubtitleAddForeignAudioSearch" : true,
+      "SubtitleAddForeignAudioSubtitle" : false,
+      "SubtitleBurnBDSub" : true,
+      "SubtitleBurnBehavior" : "foreign",
+      "SubtitleBurnDVDSub" : true,
+      "SubtitleLanguageList" : [
+
+      ],
+      "SubtitleTrackSelectionBehavior" : "none",
+      "Type" : 1,
+      "UsesPictureFilters" : true,
+      "UsesPictureSettings" : 2,
+      "VideoAvgBitrate" : 3000,
+      "VideoColorMatrixCode" : 0,
+      "VideoEncoder" : "x264",
+      "VideoFramerate" : "30",
```

Comment:
maybe same as the source which would be 30000/1001 or 25.

Reply:
Got it!

Reply:
I did changed the framerate to 29.97 to reflect the source framerate, but wouldn't that be 29970/1000 (and 25 for PAL), or were you referring to something else?